packages/grafana-runtime/src/services/LocationService.tsx

```diff
@@ -14,8 +14,8 @@ import { LocationUpdate } from './LocationSrv';
  */
 export interface LocationService {
   partial: (query: Record<string, any>, replace?: boolean) => void;
-  push: (location: H.Path | H.LocationDescriptor<any>) => void;
-  replace: (location: H.Path | H.LocationDescriptor<any>) => void;
+  push: (location: H.To, state?: unknown) => void;
+  replace: (location: H.To, state?: unknown) => void;
```

Comment:
Q: will this be a breaking change or are the `H.To` and `H.Path | H.LocationDescriptor<any>` aligned?

Reply:
They'd be aligned.